src/ngx_http_modsecurity_rewrite.c

```diff
@@ -51,6 +51,9 @@ ngx_http_modsecurity_rewrite_handler(ngx_http_request_t *r)
     if (ctx == NULL)
     {
         int ret = 0;
+        struct timeval start_tv;
+
+        ngx_gettimeofday(&start_tv);
```

Comment:
I suggest use `ngx_timeofday` like `request_time` or `upstream_xxx_time` in upstream module.

Reply:
Or `ngx_current_msec`.

Reply:
sorry, ngx_current_msec is cached. just use ngx_gettimeofday

Reply:
I'd like to use clock_gettime with CLOCK_MONOTONIC for ns.